src/hooks/logout.js

```diff
@@ -10,3 +10,4 @@ export function useLogOutDispatch() {
         resetAllState,
     };
 }
+export default useLogOutDispatch;
```

Comment:
You shouldn't shy away from default exporting functions and objects on the same line you initialize them